src/nextjournal/clerk/viewer.cljc

```diff
@@ -351,7 +355,8 @@
    (let [{:as opts :keys [viewers path offset]} (merge {:offset 0} opts)
          wrapped-value (try (wrapped-with-viewer xs viewers) ;; TODO: respect `viewers` on `xs`
                             (catch #?(:clj Exception :cljs js/Error) _ex
-                              nil))
+                              (do (println _ex)
```

Comment:
adding this print helped me catch an issue that required https://github.com/nextjournal/clerk/pull/47/commits/adc43c4e4f094eaa0e4abf6c0ed824a18cee0c2a and then the https://github.com/nextjournal/clerk/pull/47/commits/6dea81e0225d08fa63fc6eccd24c6a9321754b3a follow-up